acl/src/main/java/org/apache/rocketmq/acl/common/AclClientRPCHook.java

```diff
@@ -69,7 +72,11 @@ public void doAfterResponse(String remoteAddr, RemotingCommand request, Remoting
             if (null != header) {
                 Field[] fields = fieldCache.get(header.getClass());
                 if (null == fields) {
-                    fields = header.getClass().getDeclaredFields();
+                    Set<Field> fieldList = new HashSet<Field>();
```

Comment:
`parseRequestContent` is used for signature checks, so the order of fields matters. I think a sorted set should be adopted here.

Reply:
at the end of the function, the value of the header will be put into a SortedMap. so it doesn't matter to use HashSet here.

Reply:
But the order of fields put into returning SortedMap still depends on the order of `fieldList`.